app/models/item.js

```diff
@@ -37,13 +37,26 @@ export default cloudinaryUrl.extend({
   images: hasMany("image", { async: true }),
 
   isDispatched: Ember.computed.bool("sentOn"),
-  isDesignated: Ember.computed.bool("designation"),
   orderCode: Ember.computed.alias("designation.code"),
   updatedAt: attr("date"),
 
   imageUrl: Ember.computed.alias("image.imageUrl"),
   designateFullSet: Ember.computed.localStorage(),
 
+  isDesignated: Ember.computed(
+    "ordersPackages",
+    "ordersPackages.[]",
+    "ordersPackages.@each.state",
+    function() {
+      return (
+        this.get("ordersPackages")
+          .rejectBy("state", "requested")
```

Comment:
Will it be better to use filterBy?

```
this.get("ordersPackages").filterBy('state', 'designated').get("length") > 0
```

https://api.emberjs.com/ember/2.15/classes/Ember.Array/methods/filterBy?anchor=filterBy

Reply:
Yeah, it's actually the name that's wrong here. meant to behave as `isDesignatedOrDispatched`, I'll update it

Reply:
Actually I'll keep a proper `isDesignated` and make `isUnavailable`, makes more sense

Reply:
@Darshan-Chauhan updated 👍